source/iface/platform.rst

```diff
@@ -93,23 +96,15 @@ See `platform-example`_.
 
 .. _platform-has_extension:
 
-``has_extension``
```

Comment:
Maybe we should mention deprecated API here (because it is mentioned in the specification)?
But I'm not sure, we need to ask guys from WG

Reply:
Created issue and waiting response : https://github.com/KhronosGroup/SYCL_Reference/issues/66